rpcs3/Emu/CPU/Hypervisor.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <util/types.hpp>
+
+namespace rpcs3
+{
+	union alignas(16) hypervisor_context_t

```

Comment:
Why not #ifdef? It's going to make x86 codegen a bit worse due to useless data filling low-offset area

Reply:
Done.